src/Infrastructure/ManifestValidator/TaskManifestConfiguration.php

```diff
@@ -247,7 +244,7 @@ protected function addPlaceholderDefinition(NodeBuilder $placeholder): void
                 ->isRequired()
                 ->info('Placeholder has `/^%[a-zA-Z-_]+%$/` format.')
                 ->validate()
-                    ->ifTrue(function (string $value) {
+                    ->ifTrue(function ($value) {
```

Comment:
`preg_match()` requires second parameter to be string. Additional check or cast to string needed